tools/crm_resource_runtime.c

```diff
@@ -1308,7 +1309,9 @@ cli_resource_restart(pcmk__output_t *out, pe_resource_t *rsc, const char *host,
     }
 
     rsc_id = strdup(rsc->id);
-    if ((pe_rsc_is_clone(rsc) || pe_bundle_replicas(rsc)) && host) {
+    lookup_id = clone_strip(rsc->id);
+
+    if ((pe_rsc_is_clone(rsc) || pe_rsc_is_clone(parent) || pe_bundle_replicas(rsc)) && host) {
```

Comment:
again uber_parent(primitive)==primitive means checking is_clone(rsc) is unnecessary